docs/pages/enterprise/workflow/resource-requests.mdx

```diff
@@ -26,6 +26,13 @@ available in Teleport Enterprise.
 
 (!docs/pages/includes/tctl.mdx!)
 
+<Admonition type="warning">
+All Teleport Nodes in the cluster must be running Teleport `v10.0.0` or greater
```

Comment:
Does this apply to other services like the Database Service, Kubernetes Service, etc.?

Reply:
Yes I wasn't sure what language we are using to refer to all of those now, is it Node or Agent or something like that?

Reply:
Sadly we haven't really settled on a new term, though I've been trying to stay away from "agent" since some prospective users see that as meaning one `teleport` instance per host. It might make sense to say something like "All Teleport instances" or "All `teleport` and `tbot` instances" if that's accurate.

Reply:
Updated to "All `teleport` instances"